composer.json

```diff
@@ -11,7 +11,7 @@
     "require": {
         "silverstripe/framework": "~3.1",
         "silverstripe/cms": "~3.1",
-        "silverstripe-australia/gridfieldextensions": "~1.0.0"
+	"silverstripe-australia/gridfieldextensions": "1.1.0"
```

Comment:
If you want it to remain compatible, I'd change this to `^1.1.0`, which means that any sem-ver respecting version will continue to work. E.g. 1.2.0 but not 2.0.0